src/extension.ts

```diff
@@ -128,6 +129,10 @@ export function activate(context) {
         registerCommand('extension.draftCreate', execDraftCreate),
         registerCommand('extension.draftUp', execDraftUp),
 
+        registerCommand('extension.draftCreateDebug', execDraftCreateDebug),
```

Comment:
Suggest we keep all the Draft commands in one block, so remove blank line above?